save-backend/src/main/resources/application-prod.properties

```diff
@@ -0,0 +1 @@
+database.url=192.168.0.250:3306
```

Comment:
How will we load secrets into spring in prod?

Reply:
Made writing to a property file